pkg/operator/api/userconfig/compute.go

```diff
@@ -45,82 +45,84 @@ type SparkCompute struct {
 	MemOverheadFactor   *float64  `json:"mem_overhead_factor" yaml:"mem_overhead_factor"`
 }
 
```

Comment:
Can you initialize the `&cr.StructValidation` as a top-level `var`, since this will never change and also moving it there will catch any bugs earlier?